daml-lf/encoder/src/test/scala/com/digitalasset/daml/lf/archive/testing/EncodeV1Spec.scala

```diff
@@ -22,10 +22,7 @@ class EncodeV1Spec extends AnyWordSpec with Matchers with TableDrivenPropertyChe
   import EncodeV1Spec._
 
   val defaultParserParameters: ParserParameters[this.type] =
-    ParserParameters(
-      pkgId,
-      LanguageVersion.StableVersions.max,
-    )
+    ParserParameters(pkgId, LanguageVersion.v1_dev)
```

Comment:
I guess defaulting to dev is fine since this is only a test? maybe still worth being explicit about where we need it?

Reply:
This is a test with quick feedback for the parser.  Handle only 1.dev is fine.
Test of stable version are covered by  `//daml-lf/encoder:validate-dar-x.y` tests.